ex04.ex

```diff
@@ -55,7 +56,10 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  #def min . . . "your code"
+  def min(l) do
```

Comment:
## -1

I'd like to see code with no if statements:

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a,  b) when a < b, do: a
  def min_of(_a, b), do: b
~~~